src/burnchains/mod.rs

```diff
@@ -124,10 +132,14 @@ impl BurnchainParameters {
             chain_name: "bitcoin".to_string(),
             network_name: BITCOIN_TESTNET_NAME.to_string(),
             network_id: BITCOIN_NETWORK_ID_TESTNET,
-            first_block_height: BITCOIN_FIRST_BLOCK_TESTNET,
-            first_block_hash: FIRST_BURNCHAIN_BLOCK_HASH_TESTNET.clone(),
             stable_confirmations: 7,
             consensus_hash_lifetime: 24,
+            first_block_height: 1891063,
```

Comment:
Can you make these `const` values instead of hard-coded values?  This applies to `first_block_height`, `first_block_hash`, and `first_block_timestamp`.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1929/commits/6225b4ab7526b9a8454f1efcd759cd345fb6ce22